athena-federation-integ-test/README.md

```diff
@@ -145,9 +173,13 @@ locally from the terminal.
 The following commands should be sent after cloning the Federation GitHub repository for
 the first time, and each time the connector's code changes:
 
-1. Build the SDK and connector (SDK only needs to build once): `mvn clean install`
-2. Export the IAM credentials for the AWS account used for testing purposes.
-3. Package the connector (from the connector's directory):
+1. From the **athena-federation-sdk** dir, run `mvn clean install` if you haven't done so already.
+2. From the **athena-federation-integ-test** dir, run `mvn clean install` if you haven't done so already.
+3. From your connector's dir, run `mvn clean install`.
+4. Export the IAM credentials for the AWS account used for testing purposes. The account's Athena console should have
+   `FederationIntegrationTests` Workgroup defined and set to `Athena engine version 2` (for more information on the
```

Comment:
Is the WorkGroup name configurable? If not, it should be.

Reply:
Can be added as a parameter to the config file with a default value.